src/phoneField/PhoneField.style.tsx

```diff
@@ -1,19 +1,32 @@
 import styled from 'styled-components'
 
 import { color, inputBorderSize, radius, selectHeight, space, transition } from '../_utils/branding'
+import { normalizeHorizontally } from '../layout/layoutNormalizer'
 
 export const StyledPhoneField = styled.div`
   & .kirk-phoneField-wrapper--inline {
+    ${normalizeHorizontally};
+    margin-top: ${space.m};
+    margin-bottom: ${space.m};
```

Comment:
Not sure what the output here, but if we want to start normalizing vertically we need to make sure to use padding on a component container instead of margins, otherwise we'll get odd results because of the margin collapsing rule in CSS

Reply:
Yeah, I agree, but this one is inside a sub element. Shouldn't it be ok?

Reply:
Ah ok, you're right if these margins are scoped in this component (and they don't leak to the outside of the component) then you know what can interact with it, it's ok yes.